docs/1-introduction/1-1-getting-started/1-1-3-hello-triangle.md

```diff
@@ -349,22 +349,39 @@ constexpr D3D11_INPUT_ELEMENT_DESC vertexInputLayoutInfo[] =
         0,
         DXGI_FORMAT::DXGI_FORMAT_R32G32B32_FLOAT,
         0,
-        offsetof(VertexPositionColor, color),
+        offsetof(VertexPositionColor, color) /*resolves to 12*/,
         D3D11_INPUT_CLASSIFICATION::D3D11_INPUT_PER_VERTEX_DATA,
         0
     },
 };
 ```
 
-Now let's make sense of what each field means, let's go over one by one in order.
+**Now let's make sense of why we have the following layout:**
+(we will refer to 1 element of D3D11_INPUT_ELEMENT_DESC as a field)
 
-- `SemanticName`: lets us refer to a particular field given the string after the colon in HLSL (remember `float3 position: POSITION`)
+D3D11_INPUT_ELEMENT_DESC vertexInputLayoutInfo[] is an array since when we pass data to a GPU to read in the **vertex shader** we need to specify to the GPU how 1 vertex of data will be formatted for when we later load many vertices onto the GPU to render a triangle.
+
+**now lets explain what makes up a Input Layout Description**
+```c++
+    {
+        LPCSTR SemanticName,
+        UINT SemanticIndex,
+        DXGI_FORMAT Format,
+        UINT InputSlot,
+        UINT AlignedByteOffset,
+        D3D11_INPUT_CLASSIFICATION InputSlotClass,
+        UINT InstanceDataStepRate
+    }
+```
+- `SemanticName`: Tells the GPU to expect in the **vertex shader** that a variable in a vertex shader input struct (`VSInput` in this case) with the same `SemanticName` (string after a colon in the vertex shader's declaration — recall `float3 position: STUFF_AFTER_COLON`) will be where the GPU puts the received data.
```

Comment:
This sentence is quite long and too many stuff is wrapped in braces somehow :) can this maybe broken into 2 or more sentences and dont be afraid to write more text if you have to.

the last (string after a colon ... feels kinda strange the way its written, but it could also just be me

Reply:
I agree, I adjusted it and will make a more natural language setup to explain the process.

I start with what it is, how it links - and then simply a part of where it actually is shown in the gpu code, but revised the English to be better